server/src/ClientToDBAPI/IntersectionsRoute/mainRoute.ts

```diff
@@ -84,14 +84,14 @@ const convertDBEvent = (event: ContactEvent) => {
     };
 }
 
-intersectionsRoute.get('/contactEvent/:investigationId', (request: Request, response: Response) => {
+intersectionsRoute.get('/contactEvent/:investigationId/:minimalDateToFilter', (request: Request, response: Response) => {
     const contactEventLogger = logger.setup({
         workflow: `query investigation's contact events`,
         user: response.locals.user.id,
         investigation: response.locals.epidemiologynumber
     });
 
-    const parameters = {currInvestigation: Number(request.params.investigationId)};
+    const parameters = {currInvestigation: Number(request.params.investigationId), minimalDateToFilter: new Date(request.params.minimalDateToFilter)};
```

Comment:
why new Date for the second time?
You already did it in useInteractionTab

Reply:
I must keep the date here due to a graphql error, but I changed the sent value to be a string